catalog/spatial/registry/registry-identification-plugin/src/main/java/org/codice/ddf/registry/identification/IdentificationPlugin.java

```diff
@@ -226,25 +319,49 @@ private String getRegistryId(Metacard mcard) {
                 .toString();
     }
 
-    public void init() throws FederationAdminException {
-
-        List<Metacard> registryMetacards = federationAdmin.getRegistryMetacards();
-
+    public void init()
+            throws UnsupportedQueryException, SourceUnavailableException, FederationException {
+
+        List<Metacard> registryMetacards;
+        Filter registryFilter = filterBuilder.attribute(Metacard.TAGS)
+                .is()
+                .like()
+                .text(RegistryConstants.REGISTRY_TAG);
+        Subject systemSubject = Security.getInstance()
+                .getSystemSubject();
+        QueryImpl query = new QueryImpl(registryFilter);
```

Comment:
Definitely doesn't look great, right?
This was a way to address an issue where the identity metacard would not be returned in the first "page" of the query for the default page size of 20. I know Phil, Gordo and Chris all looked at this with me and may be able to weigh in a bit more as to how horrible it would be to leave it at 1000 for now. Could make a constant if that would be better.